views/users/landing.html

```diff
@@ -6,34 +6,36 @@
 
 {% block header %}
   {{ super() }}
-  <h1>{{title}}</h1>
 {% endblock %}
 
 {% block main %}
+<h1>{{title}}</h1>
 <p><a href="{{config.base}}/{{config.actions.add}}">{{config.base}} | {{config.actions.add}}</a></p>
 <h2>Content</h2>
 
-{% if users %}
-  <table>
-    <tr><th>email</th><th>role</th><th>{{config.actions.edit}}</th><th>{{config.actions.delete}}</th><th>date</th></tr>
-  {% for u in users %}
-    {% if u.email %}
-      <tr>
-        <td>{{u.email}}</td>
-        <td>{{u.role}}</td>
-        <td><a href="{{config.base}}/{{u.id}}/{{config.actions.edit}}">{{config.actions.edit}}</a></td>
-        <td>
-        {% if u.id != user.id %}
-        <a href="{{config.base}}/{{u.id}}/{{config.actions.delete}}">{{config.actions.delete}}</a>
-        {% endif %}
-        </td>
-        <td>{{u.created}}</td>
-      </tr>
+  <div class="table-fluid">
```

Comment:
@rachelnicole so we are using a table for the content landing pages for managing content, but this one is changed to div's. We need to choose one or the other. Did you make this html change to use flexbox?

@Snugug is content like this (management pages with lists of users or content + actions (like edit/delete)) considered tabular data legitimate to use a table?